internal/services/healthcare/healthcare_apis_workspace_resource.go

```diff
@@ -0,0 +1,143 @@
+package healthcare
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/services/healthcareapis/mgmt/2021-11-01/healthcareapis"
+	"github.com/hashicorp/terraform-provider-azurerm/helpers/azure"
+	"github.com/hashicorp/terraform-provider-azurerm/helpers/tf"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/healthcare/parse"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/healthcare/validate"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tags"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/timeouts"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+)
+
+func resourceHealthcareApisWorkspace() *pluginsdk.Resource {
+	return &pluginsdk.Resource{
+		Create: resourceHealthcareApisWorkspaceCreateUpdate,
+		Read:   resourceHealthcareApisWorkspaceRead,
+		Update: resourceHealthcareApisWorkspaceCreateUpdate,
```

Comment:
even create and update is using the same function?

Reply:
Yep, even if the function call is the same for both create and update we should be splitting them so that `ignore_changes` works.

But it looks like this API also supports just an `Update` i.e. `PATCH`?

Reply:
The update function only supports Tags update, I replaced the origin createUpdate with the tag update